developing_packages/package_layout.rst

```diff
@@ -215,6 +215,13 @@ In the ``layout()`` method you can set:
            (e.g the `xx-config.cmake` files from the ``CMakeDeps``)
          - **self.folders.imports**: To specify a subfolder where to write the files copied when using the ``imports(self)``
            method in a ``conanfile.py``.
+         - **self.folders.root**: To specify the relative path from the ``conanfile.py`` to the root of the project, in case 
+           the conanfile.py is in a subfolder and not in the project root. If defined all the other paths will be relative to
+           the project root, not to the location of the ``conanfile.py``
+         - **self.folders.base_source_folder** points to the root folder that is used as a base to compute the actual ``self.source_folder``.
```

Comment:
I would extract this one from the list because it is read-only (we should say it) and previously the docs say: `In the ``layout()`` method you can set:`

Reply:
Agree

Reply:
Fixed